ddl/ingest/backend_mgr.go

```diff
@@ -121,7 +121,7 @@ func createLocalBackend(ctx context.Context, cfg *Config) (*local.Backend, error
 	regionSizeGetter := &local.TableRegionSizeGetterImpl{
 		DB: nil,
 	}
-	backendConfig := local.NewBackendConfig(cfg.Lightning, int(LitRLimit), cfg.KeyspaceName)
+	backendConfig := local.NewBackendConfig(cfg.Lightning, int(LitRLimit), cfg.KeyspaceName, "")
```

Comment:
maybe ask for a DDL developer to review the value

Reply:
+ @wjhuang2016  PTAL

Reply:
LGTM~